hedera-node/configuration/previewnet/application.properties

```diff
@@ -1 +1,9 @@
+# Except for network-specific properties (e.g. previewnet vs. mainnet), this file is intended to be an
+# exact replica of file 121 on testnet and should match bootstrap.properties exactly.
+# It will be used by modular code for all the property overrides.
+# It does not have to match bootstrap.properties once that file is deprecated by the switch from mono-service to hedera-config.
 
+#Overrides that differ based on the network
+bootstrap.genesisPublicKey=c249a323c878f5b5e2daccda6d731e6fdc32f870228d1cd4fae559d947dbc36c
+contracts.chainId=297
+ledger.id=0x02
```

Comment:
Added comments similar to `testnet/application.properties`. 
Removed `contracts.evm.version.dynamic` as requested by the smart contract team.
Otherwise, this file is an exact copy of the sibling `bootstrap.properties` file (minus the properties that match mainnet).